mat64/bits_amd64.s

```diff
@@ -0,0 +1,24 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//+build !noasm
+
+// TODO(kortschak): use textflag.h after we drop Go 1.3 support
+//#include "textflag.h"
+// Don't insert stack check preamble.
+#define NOSPLIT	4
+
+// func bits(v uint) (n byte)
```

Comment:
Yeah, that's fine. I did a check of the first 1e6 values to see if it matches a naive Go inplementation. I think the asm is clearer than the Go code I have to do the same thing, given that it just literally does the operations that are obvious.